tests/algorithms/test_sqil.py

```diff
@@ -137,12 +219,53 @@ def test_sqil_performance(
     )
 
 
+def test_sqil_performance_discrete(
+    rng: np.random.Generator,
+    pytestconfig: pytest.Config,
+    cartpole_venv: vec_env.VecEnv,
+):
+    _test_sqil_performance(
+        rng,
+        pytestconfig,
+        cartpole_venv,
+        "seals/CartPole-v0",
+        rl_algo_class=dqn.DQN,
+        rl_kwargs=dict(
+            learning_starts=500,
+            learning_rate=0.002,
+            batch_size=220,
+            seed=42,
+        ),
+    )
+
+
+# TODO: make it work reliably for other algorithms besides SAC.
```

Comment:
That would be nice ;) Perfectly possible some of the other algorithms are just less sample efficient though and might be hard to get significant improvement in a short unit test.

Reply:
Ok, with a bit of tuning (lr / gamma) I think I got it to work.